tests/integration/conftest.py

```diff
@@ -328,10 +333,13 @@ async def fixture_registry_async_client(
     if registry_url:
         client = Client(server_uri=registry_url)
     else:
-        client_factory = await aiohttp_client(registry_async.app)
-        client = Client(client=client_factory)
 
-    with closing(client):
```

Comment:
was this removed because closing client is async? Because there is an async version of closing which would be better suited: https://docs.python.org/3/library/contextlib.html#contextlib.aclosing

Reply:
Yes, it's because closing client is async.
aclosing would be better but it's python >= 3.10.

Reply:
@kmichel-aiven fair point, I didn't look at the supported version.